CHANGES.rst

```diff
@@ -1,6 +1,20 @@
 Changes
 =======
 
+Version 0.5.0 (2019-04-23)
+--------------------------
+
```

Comment:
General comments: start with user-oriented features, finish with developer-oriented features; formulate things in "positive tense" instead of "negative tense"; join news items together when reasonable; put variables last rather than first in phrases.  Something like:

```
- Separates cluster infrastructure pods from runtime workflow engine pods that will be created by workflow controller.
- Adds support for exposing user interactive sessions such as Jupyter notebook via Traefik ingress controller.
- Introduces configurable CVMFS and CephFS shared volume mounts. Changes ``SHARED_VOLUME_PATH`` to ``/var/reana/``.
- Adds support for optional HTTPS protocol termination at the ``REANA-Server`` component.
- Improves workflow execution queuing and scheduling via ``REANA-Server`` sidecar.
- Removes unused ``REANA-Workflow-Monitor`` component and ``ZeroMQ`` service.
- Enables Flask debugging mode for the cluster development configuration by setting ``FLASK_ENV`` accordingly.
```

(... but please recheck...)

Reply:
BTW one more thing:  I think we are missing a news item about versions, such as:

```
- Upgrades to Kubernetes 1.14, Helm 2.13 and Minikube 1.0.
```

Could be one of the leading items, as was the case in v0.3.1 for example, or one of the very last news item too...